design_state.json

```
@@ -2,7 +2,7 @@
   "format_version": "1.3",
   "design_name": "phase5_soc",
   "created_at": "2026-05-31T00:00:00Z",
-  "updated_at": "2026-06-01T07:55:00+07:00",
+  "updated_at": "2026-06-01T08:40:00+07:00",
```

Biome (2.4.15)

[error] The key updated_at was already declared.

(lint/suspicious/noDuplicateObjectKeys)